assets/less/makeup.less

```diff
@@ -0,0 +1,178 @@
+.night-mode .makeup .l {color: #f99b15; } /* :literal */
+.night-mode .makeup .ld {color: #776e71; } /* :literal_date */
+.night-mode .makeup .m {color: #f99b15; } /* :number */
+.night-mode .makeup .mb {color: #f99b15; font-weight: normal } /* :number_bin */
+.night-mode .makeup .mf {color: #f99b15; font-weight: normal } /* :number_float */
+.night-mode .makeup .mh {color: #f99b15; font-weight: normal } /* :number_hex */
+.night-mode .makeup .mi {color: #f99b15; font-weight: normal } /* :number_integer */
+.night-mode .makeup .mo {color: #f99b15; font-weight: normal } /* :number_oct */
+.night-mode .makeup .n {color: #e7e9db; } /* :name */
+.night-mode .makeup .na {color: #06b6ef; } /* :name_attribute */
+.night-mode .makeup .nb {color: #e7e9db; } /* :name_builtin */
+.night-mode .makeup .nc {color: #fec418; } /* :name_class */
+.night-mode .makeup .nd {color: #5bc4bf; } /* :name_decorator */
+.night-mode .makeup .ne {color: #ef6155; } /* :name_exception */
+.night-mode .makeup .nf {color: #06b6ef; } /* :name_function */
+.night-mode .makeup .ni {color: #ef6155; } /* :name_entity */
+.night-mode .makeup .nl {color: #e7e9db; } /* :name_label */
+.night-mode .makeup .nn {color: #fec418; } /* :name_namespace */
+.night-mode .makeup .no {color: #ef6155; } /* :name_constant */
+.night-mode .makeup .nt {color: #5bc4bf; } /* :name_tag */
+.night-mode .makeup .nv {color: #ef6155; } /* :name_variable */
+.night-mode .makeup .nx {color: #06b6ef; } /* :name_other */
+.night-mode .makeup .o {color: #5bc4bf; font-weight: normal } /* :operator */
+.night-mode .makeup .ow {color: #5bc4bf; font-weight: normal } /* :operator_word */
+.night-mode .makeup .p {color: #e7e9db; } /* :punctuation */
+.night-mode .makeup .py {color: #e7e9db; } /* :name_property */
+.night-mode .makeup .s {color: #48b685; } /* :string */
+.night-mode .makeup .s1 {color: #48b685; } /* :string_single */
+.night-mode .makeup .s2 {color: #48b685; } /* :string_double */
+.night-mode .makeup .sa {color: #48b685; } /* :string_affix */
+.night-mode .makeup .sb {color: #48b685; } /* :string_backtick */
+.night-mode .makeup .sc {color: #e7e9db; } /* :string_char */
+.night-mode .makeup .sd {color: #776e71; } /* :string_doc */
+.night-mode .makeup .se {color: #f99b15; } /* :string_escape */
+.night-mode .makeup .sh {color: #48b685; } /* :string_heredoc */
+.night-mode .makeup .si {color: #f99b15; } /* :string_interpol */
+.night-mode .makeup .sr {color: #48b685; } /* :string_regex */
+.night-mode .makeup .ss {color: #48b685; } /* :string_symbol */
+.night-mode .makeup .sx {color: #48b685; } /* :string_other */
+.night-mode .makeup .sx {color: #48b685; } /* :string_sigil */
+.night-mode .makeup .vc {color: #ef6155; } /* :name_variable_class */
+.night-mode .makeup .vg {color: #ef6155; } /* :name_variable_global */
+.night-mode .makeup .vi {color: #ef6155; } /* :name_variable_instance */
+.night-mode .makeup .vm {color: #ef6155; } /* :name_variable_magic */
```

Comment:
I think if these classes are grouped they could be simplified

Reply:
What do you mean by "grouped"? Something like:

```css
.night-mode .makeup .vc, .vg, .vi, .vm {color: #ef6155; }
```

or simply replacing them by a single class? If so, then that's a definite No, because even if this particular theme doesn't show a distinction between these classes, other themes might.

Reply:
I mean nesting classes